docs/catalogs/fileset/gvfs/index.md

```diff
@@ -0,0 +1,1078 @@
+    and then set this configuration to the class name to inject custom code.
+  </td>
+  <td>`org.apache.gravitino.filesystem.hadoop.NoOpHook`</td>
+  <td>No</td>
+  <td>`0.9.0-incubating`</td>
+</tr>
+<tr>
+  <td><tt>fs.gravitino.client.request.header.</tt></td>
+  <td>
+    The configuration key prefix for the Gravitino client request header.
+    You can set the request header for the Gravitino client.
+  </td>
+  <td>(none)</td>
+  <td>No</td>
+  <td>`0.9.0-incubating`</td>
+</tr>
+<tr>
+  <td><tt>fs.gravitino.enableCredentialVending</tt></td>
+  <td>Whether to enable credential vending for the Gravitino Virtual File System. </td>
+  <td>`false`</td>
+  <td>No</td>
+  <td>`0.9.0-incubating`</td>
+</tr>
+</tbody>
+</table>
+
+In addition to the above properties, to access fileset like S3, GCS, OSS and custom fileset,
+some extra properties are needed. For more information, please see 
+
+- [S3 GVFS Java client configurations](../hadoop/s3.md#using-the-gvfs-java-client-to-access-the-fileset)
+- [GCS GVFS Java client configurations](../hadoop/gcs.md#using-the-gvfs-java-client-to-access-the-fileset)
+- [OSS GVFS Java client configurations](../hadoop/oss.md#using-the-gvfs-java-client-to-access-the-fileset)
+- [Azure Blob Storage GVFS Java client configurations](../hadoop/adls.md#using-the-gvfs-java-client-to-access-the-fileset)
+
+#### Custom fileset 
+
+Since *0.7.0-incubating*, users can define their own fileset type and configure the corresponding properties.
+For more details, please refer to [Custom Fileset](../hadoop/hadoop-catalog.md#how-to-custom-your-own-hcfs-file-system-fileset).
+If you want to access the custom fileset through GVFS, you need to configure the corresponding properties.
+
+<table>
+<thead>
+<tr>
+  <td>Configuration item</td>
+  <td>Description</td>
+  <td>Default value</td>
+  <td>Required</td>
+  <td>Since version</td>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><tt>your-custom-properties</tt></td>
+  <td>
+    The properties to be used to create a FileSystem instance
+    in `CustomFileSystemProvider#getFileSystem`
+  </td>
+  <td>(none)</td>
+  <td>No</td>
+  <td></td>
+</tr>
+</tbody>
+</table>
+ 
+You can configure these properties in two ways:
+
+1. Before obtaining the `FileSystem` in the code, construct a `Configuration` object and set its properties:
+
+   ```java
+   Configuration conf = new Configuration();
+   conf.set("fs.AbstractFileSystem.gvfs.impl", "org.apache.gravitino.filesystem.hadoop.Gvfs");
+   conf.set("fs.gvfs.impl", "org.apache.gravitino.filesystem.hadoop.GravitinoVirtualFileSystem");
+   conf.set("fs.gravitino.server.uri", "http://localhost:8090");
+   conf.set("fs.gravitino.client.metalake", "mymetalake");
+   Path filesetPath = new Path("gvfs://fileset/mycatalog/myschema/my_fileset_1");
+   FileSystem fs = filesetPath.getFileSystem(conf);
+   ```
+
+1. Configure the properties in the `core-site.xml` file for the Hadoop environment:
```

Comment:
The number should be 2.

Reply:
In Markdown, an ordered list doesn't have to have an explicit number. The general practice is to let the renderer figure out which one should be number 2 and which one is number 3. The benefit is that we, as the docs writers, don't need to manually maintain the order of the list. In other words, if you want to insert an item into a list, you focus on that new item without worrying about the numbers after the one you insert. Similarly, when you remove an unnecessary item from a list, you don't need to change other items. The markdown parser (a correctly working one of course) will take care of it.

Hope this solves your concern.

Reply:
Got it. Thanks for explaining